### PR TITLE
adding steps to recover cluster

### PR DESCRIPTION
**Pull request by warroyo**
/kind documentation

**What this PR does / why we need it**:

This adds the steps to recover a management cluster that has lost it's load balancer.

### CODE REVIEW

File: docs/book/src/topics/troubleshooting.md [hunk 1]
```diff
@@ -59,3 +59,136 @@ $ aws iam get-instance-profile --instance-profile-name control-plane.cluster-api
...
+    * Replace the `server` with `server: https://localhost:6443`
+
+    * Add `insecure-skip-tls-verify: true`
+
+    * Comment out `certificate-authority-data:`
+
+2. Export the kubeconfig and ensure you can connect
+
+    ```bash
+    export KUBECONFIG=/etc/kubernetes/admin.conf`
```
[the remark this answers is not in the document]
Comment (warroyo) [reply docs/book/src/topics/troubleshooting.md]: thanks @shivi28 this is fixed

File: docs/book/src/topics/troubleshooting.md [hunk 2]
```diff
@@ -59,3 +59,136 @@ $ aws iam get-instance-profile --instance-profile-name control-plane.cluster-api
...
+
+
+### **Get rid of the lingering duplicate cluster**
+
+1. since there is a duplicate cluster that is trying to be deleted and can't due to some resources being unable to cleanup since they are in use we need to stop the conflicting reconciliation process. Edit the duplicate aws cluster object and remove the `finalizers`
+
+    ```bash
+    kubectl edit awscluster <clustername>
+    ```
+2. `kubectl get clusters` to verify it's gone
```
[the remark this answers is not in the document]
Comment (warroyo) [reply docs/book/src/topics/troubleshooting.md]: I just updated this to add the describe. let me know if this looks good now @shivi28

File: docs/book/src/topics/troubleshooting.md [hunk 3]
```diff
@@ -59,3 +59,136 @@ $ aws iam get-instance-profile --instance-profile-name control-plane.cluster-api
...
+    ```bash
+    kubectl edit secret -n <namespace> <cluster-name>-kubeconfig`
+    ```
+6. At this point things should start to reconcile on their own, but we can use the commands in the next step to force it. 
+
+
+### Roll all of the nodes to make sure everything is fresh
+
+1. `kubectl patch kcp <clusternamekcp> -n namespace --type merge -p "{\"spec\":{\"rolloutAfter\":\"`date +'%Y-%m-%dT%TZ'`\"}}"`
+
```
[the remark this answers is not in the document]
Comment (warroyo) [reply docs/book/src/topics/troubleshooting.md:199]: just updated this one it should be formatted properly now. @shivi28

**Comment by warroyo** [comment]
@sedefsavas @shivi28 is there anything left pending here to get merged?

File: docs/book/src/topics/troubleshooting.md [hunk 4]
```diff
@@ -59,3 +59,144 @@ $ aws iam get-instance-profile --instance-profile-name control-plane.cluster-api
...
+2. Export the kubeconfig and ensure you can connect
+
+    ```bash
+    export KUBECONFIG=/etc/kubernetes/admin.conf
+    kubectl get nodes
+    ```
+
+
+### **Get rid of the lingering duplicate cluster**
```
Comment (sedefsavas) [inline docs/book/src/topics/troubleshooting.md:84]: This is not explicit, would be good to explain how you end up in this situation right after `## Recover a management cluster after losing the api server load balancer` title.

Also, there is no need to have a lingering Cluster, this workaround works for accidentally manually deleting the api-server LB.

Comment (warroyo) [reply docs/book/src/topics/troubleshooting.md:89]: Thanks, I made some updates. I think it's important to keep the lingering cluster section but I put a statement at the start to explain that it's only needed in a certain scenario

**Comment by sedefsavas** [comment]
LGTM
Could you squash the commits @warroyo before merging.

**Pull request merged**